.github/workflows/release.yaml

```diff
@@ -238,3 +250,80 @@ jobs:
             ./build/*.deb
             ./build/*.rpm
           retention-days: 7
+
+  publish-winget:
+    runs-on: windows-latest
+    needs: release
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      # If the event that triggered the build was an annotated tag (which our
+      # tags are supposed to be), actions/checkout has a bug where the tag in
+      # question is only a lightweight tag and not a full annotated tag. This
+      # command seems to fix it.
+      # https://github.com/actions/checkout/issues/290
+      - name: Fetch git tags
+        run: git fetch --tags --force
+
+      - name: Install wingetcreate
+        run: |
+          Invoke-WebRequest https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
+
+      - name: Submit updated manifest to winget-pkgs
+        run: |
+          # The package version is the same as the tag minus the leading "v".
+          # The version in this output already has the leading "v" removed but
+          # we do it again to be safe.
+          $version = "${{ needs.release.outputs.version }}".Trim('v')
+
+          $release_assets = gh release view --repo coder/coder "v${version}" --json assets | `
```

Comment:
Pro tip: `gh` builtin `jq` is pretty neat in these situations:

```shell
gh release view --repo coder/coder v0.14.2 --json assets --jq '.assets[] | select(.name | test("_windows_amd64_installer.exe$")) | .url'
```

TBH the pwsh script is pretty neat too.